gulp/utils/configHelpers.js

```diff
@@ -0,0 +1,22 @@
+const glob = require('glob');
+
+module.exports = {
+  getAvailableLanguages: (config) => {
+    const entry = config.entryGlobs.content;
+    const entryPath = config.entryPaths.content;
+    return entry.map(e => glob.sync(e))
+      .reduce((a, b) => a.concat(b), [])
```

Comment:
I though about this, but this is a simple array merge function. I don't think naming this would make it more helpful:

```
(a,b) => a.concat(b)
```
```
(mergedPathsAcc, currentPaths) => mergedPathsAcc.concat(currentPaths)
```